spk/syncthing-inotify/Makefile

```diff
@@ -0,0 +1,37 @@
+SPK_NAME = syncthing-inotify
+SPK_VERS = 0.8.2
+SPK_REV = 1
+SPK_ICON = src/${SPK_NAME}.png
+BETA = 1
+
+UNSUPPORTED_ARCHS = $(PPC_ARCHES)
+
+DEPENDS = cross/busybox cross/$(SPK_NAME)
+SPK_DEPENDS = syncthing
+START_DEP_SERVICES = syncthing
+
+MAINTAINER = SynoCommunity
+DESCRIPTION  = Syncthing uses a rescan interval to detect changes in folders. Syncthing-inotify uses OS primitives to detect changes as soon as they happen. Therefore, near real-time synchronisation can be achieved.
+RELOAD_UI = no
+DISPLAY_NAME = Syncthing-Inotify
+CHANGELOG = "Initial release"
+
+HOMEPAGE   = https://github.com/syncthing/syncthing-inotify
+LICENSE    = MPL-2.0
+
+INSTALLER_SCRIPT = src/installer.sh
+SSS_SCRIPT       = src/dsm-control.sh
+
+INSTALL_PREFIX = /usr/local/$(SPK_NAME)
+
+POST_STRIP_TARGET = inotify_extra_install
+
+BUSYBOX_CONFIG = daemon nice
```

Comment:
See earlier comment, if the busybox dep can be removed, these two lines can go too.